README.md

```diff
@@ -1,6 +1,31 @@
-# 10x
+# 10x website Problem Statement and Goals
+
+### [Macro]
```

Comment:
Heading levels should kind of act like an outline indent and shouldn't skip levels. That is, each page should have one h1, and the next level down should be an h2 (or two ## in markdown). 

So either change the Macro header to be H2, or make it a subheader in a section that starts with an h2. 

Also stylistically I don't think we need the square brackets, it looks a little strange.